spec/requests/api/providers_spec.rb

```diff
@@ -929,6 +929,52 @@ def gen_import_request
     end
   end
 
+  context 'cloud tenants subcollection' do
+    before do
+      @provider = FactoryGirl.create(:ems_openstack)
+      @cloud_tenant = FactoryGirl.create(:cloud_tenant, :ext_management_system => @provider)
+    end
+
+    it 'queries all cloud tenants' do
+      api_basic_authorize subcollection_action_identifier(:providers, :cloud_tenants, :read, :get)
+      expected = {
```

Comment:
Nit, but can you group verification steps (this `expected`) together at the bottom?